packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -196,12 +197,22 @@ class RenderEditable extends RenderBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// This is used for updating the text in the text field in the event
+  /// of cut or paste.
```

Comment:
we should give more detail. How is it used? Who should set it? What happens when it is set dynamically? Can it be null? etc